openmc/universe.py

```diff
@@ -360,15 +361,18 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
 
             # Create a figure sized such that the size of the axes within
             # exactly matches the number of pixels specified
-            px = 1/plt.rcParams['figure.dpi']
-            fig, ax = plt.subplots()
-            params = fig.subplotpars
-            width = pixels[0]*px/(params.right - params.left)
-            height = pixels[0]*px/(params.top - params.bottom)
-            fig.set_size_inches(width, height)
+            if axes is None:
+                px = 1/plt.rcParams['figure.dpi']
+                fig, axes = plt.subplots()
+                params = fig.subplotpars
+                width = pixels[0]*px/(params.right - params.left)
+                height = pixels[0]*px/(params.top - params.bottom)
+                fig.set_size_inches(width, height)
+            else:
+                pass
```

Comment:
To follow @shimwell's suggestion, you can remove the `else` and the `pass`:
```suggestion
```

Reply:
Removed